### ISSUE TITLE
Cannot spawn a pet

### ISSUE THREAD

**Issue by NOKIEQU** [creepycats/SCPCosmetics#3]
when typing a pet spawn command i get this error in a server console
```
[2023-10-31 14:40:40.772 +00:00] [ERROR] [Exiled.API] System.ArgumentException: An item with the same key has already been added. Key: System.ArraySegment`1[System.Byte]
                                   at System.Collections.Generic.Dictionary`2[TKey,TValue].TryInsert (TKey key, TValue value, System.Collections.Generic.InsertionBehavior behavior) [0x0015a] in <05089c1b6989455fb1452c627044bac4>:0
                                   at System.Collections.Generic.Dictionary`2[TKey,TValue].Add (TKey key, TValue value) [0x00000] in <05089c1b6989455fb1452c627044bac4>:0
                                   at Exiled.API.Extensions.MirrorExtensions.get_WriterExtensions () [0x00085] in <82c23c122317447280ee5275f620e42f>:0
                                   at Exiled.API.Extensions.MirrorExtensions+<>c__DisplayClass31_0.<SendFakeSyncVar>g__CustomSyncVarGenerator|0 (Mirror.NetworkWriter targetWriter) [0x0002b] in <82c23c122317447280ee5275f620e42f>:0
                                   at Exiled.API.Extensions.MirrorExtensions.MakeCustomSyncWriter (Mirror.NetworkIdentity behaviorOwner, System.Type targetType, System.Action`1[T] customSyncObject, System.Action`1[T] customSyncVar, Mirror.NetworkWriter owner, Mirror.NetworkWriter observer) [0x00082] in <82c23c122317447280ee5275f620e42f>:0
                                   at Exiled.API.Extensions.MirrorExtensions.SendFakeSyncVar (Exiled.API.Features.Player target, Mirror.NetworkIdentity behaviorOwner, System.Type targetType, System.String propertyName, System.Object value) [0x00028] in <82c23c122317447280ee5275f620e42f>:0
                                   at SCPCosmetics.Types.HatPlayerComponent+<MoveHat>d__3.MoveNext () [0x00204] in <cc236bf4cb504f358b81efd3898e8a28>:0
```
also getting this error in the client console:
![image](https://github.com/creepycats/SCPCosmetics/assets/51715318/35047a30-13af-4aad-a735-fffe15a123f3)

**Comment by creepycats** [comment]
You shouldn't be using this version. I've left SCPSL and the current version is not tested especially for the new update.

**Comment by NOKIEQU** [comment]
Is there a similar plugin out there that I can use>

**Comment by creepycats** [comment]
Not really. Ill fix it within a month if I can find a SL server I enjoy playing on and relearn to love the game.

**Comment by creepycats** [comment]
Update: Fixed for 13.3.1
